web-registry/src/lib/queries/react-query/utils/getFromCacheOrFetch.ts

```diff
@@ -0,0 +1,19 @@
+import { QueryClient, QueryObserverOptions } from '@tanstack/react-query';
+
+interface GetFromCacheOrFetchParams<QueryResponse> {
+  reactQueryClient?: QueryClient;
+  query: QueryObserverOptions<QueryResponse>;
+}
+
+export async function getFromCacheOrFetch<QueryResponse>({
```

Comment:
perhaps the name `checkCacheOrFetch` would be clearer (since nothing is returned at this point) ?

Reply:
oh right, I see..